app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -86,6 +92,21 @@ object ProfileHelper {
 		)
 	}
 
+	val max1080pProfileConditions by lazy {
+		arrayOf(
+			ProfileCondition(
+				ProfileConditionType.LessThanEqual,
+				ProfileConditionValue.Width,
+				"1920"
+			),
+			ProfileCondition(
+				ProfileConditionType.LessThanEqual,
+				ProfileConditionValue.Height,
+				"1080"
```

Comment:
Wouldn't it make more sense to use `1200` as height so 16:10 video works?

Reply:
I would have to find a video to test that. It _looks_ like 1920x1080 is the max in the Amazon docs, but it's not entirely clear imo.l

Reply:
It looks like allowing `1200` for height produces the same error about exceeding device capabilities.